gobblin-modules/gobblin-kafka-common/src/test/java/org/apache/gobblin/metrics/reporter/KafkaEventReporterTest.java

```diff
@@ -54,7 +57,7 @@ public void testKafkaEventReporter() throws IOException {
 
     MockKafkaPusher pusher = new MockKafkaPusher();
     KafkaEventReporter kafkaReporter = getBuilder(context, pusher).build("localhost:0000", "topic");
-
+    kafkaReporter.getQueueCapacity();
```

Comment:
Seems like we are ignoring the return value here. If not needed, remove this line.